samples/abn-sample/backend/go.sum

```diff
@@ -0,0 +1,15 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
For the sample, this may not be needed.

Reply:
is needed; Dockerfile does `go mod download`